.github/actions/setup-linux/action.yml

```diff
@@ -59,6 +59,12 @@ runs:
           aws ecr get-login-password --region "$AWS_DEFAULT_REGION" | docker login --username AWS \
               --password-stdin "$AWS_ACCOUNT_ID.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com"
 
+          # For LF Runners we need to make sure we also login to Meta's ECR docker registry too.
+          if [ "$AWS_ACCOUNT_ID" != "308535385114" ] ; then
```

Comment:
nit: can we turn this into a variable and reuse it below as well?

Reply:
Yes we can. I'll update the PR.